### DISCUSSION TITLE
Adeus, Internet Explorer

### DISCUSSION THREAD

**Discussion by vish-blog** [vish-blog/comments#345] [Imagens]
Post de referência: https://www.vish.blog.br/imagens/tirinhas/adeus-internet-explorer/

**Comment by one-eyed-willy** [comment]
Não lembro de já ter tido problemas com IE. Tá ele é bem mais lento que os outros, mas será que é pra tanto?

**Comment by vish-blog** [comment]
> [Willy Caolho](</perfil/?user=one-eyed-willy>) disse [segunda-feira, 20 de junho de 2022](<https://www.vish.blog.br/imagens/tirinhas/adeus-internet-explorer/#comment_1160660621>):

> Não lembro de já ter tido problemas com IE. Tá ele é bem mais lento que os outros, mas será que é pra tanto?

Como desenvolvedor, pessoalmente, o maior problema com o Internet Explorer foi a ausência de suporte às tecnologias modernas. Para dar suporte ao IE diversos *polyfills* — bibliotecas "tapa-buracos" — seriam necessárias aqui no site, o que pode torná-lo mais lento.
